File: frontend/src/app/models/GraphDTO.ts
import {Person} from './Person';
import {Group} from './Group';
import {Link} from '../d3/models/link';

export class GraphDTO {
  users: Person[];
  groups: Group[];
  links: Link[];

  constructor(users: Person[], groups: Group[], links: Link[]) {
    this.users = users;
    this.groups = groups;
    this.links = links;
  }
}

File: backend/src/main/java/com/epam/vkneo4j/queryresults/GraphDTO.java
package com.epam.vkneo4j.queryresults;

import com.epam.vkneo4j.model.Group;
import com.epam.vkneo4j.model.Person;
import java.util.List;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class GraphDTO {

  List<Person> users;
  List<Group> groups;
  List<Link> links;
}
